### ISSUE TITLE
Add returning to bash instructions

### ISSUE THREAD

**Issue by dan9186** [myzsh/myzsh#84]
Not that we want people to uninstall, but I have had a few questions from people around "how do I go back if I don't like it but want it more permanent for the time being?"  So I'm adding this for the time being as a reminder to flesh out more documentation, specifically with info around "uninstalling."

**Comment by brimstone** [comment]
Is this more than `rm - rf $MYZSH`? Do we want to alter the user's `.zshrc` or just let them know it needs hand editing?

**Comment by dan9186** [comment]
I don't believe it is anything more than that, but it seems that unless the person is super confident at unhorking their system there's a lot of hesitancy to even try it out.  As much as we can assist and as much as we can convey what needs to be done without destroying their stuff, I think is the best approach for it.
